comms/core/src/connectivity/manager.rs

```diff
@@ -659,6 +659,21 @@ impl ConnectivityManagerActor {
         let (node_id, mut new_status, connection) = match event {
             PeerDisconnected(_, node_id, minimized) => (node_id, ConnectionStatus::Disconnected(*minimized), None),
             PeerConnected(conn) => (conn.peer_node_id(), ConnectionStatus::Connected, Some(conn.clone())),
+            PeerConnectFailed(node_id, ConnectionManagerError::AllPeerAddressesAreExcluded(msg)) => {
+                warn!(
+                    target: LOG_TARGET,
+                    "Peer '{}' contains only excluded addresses ({})",
+                    node_id,
+                    msg
+                );
+                self.ban_peer(
+                    node_id,
+                    Duration::from_secs(3 * 24 * 60 * 60), // 3 days
+                    "All peer addresses are excluded (User intervention)".to_string(),
+                )
+                .await?;
+                (node_id, ConnectionStatus::Failed, None)
+            },
```

Comment:
I dont think we should ban the node. 
If the node is banned, they cannot dial us. Which they can and should be